python/cuspatial/cuspatial/core/_column/geocolumn.py

```diff
@@ -179,6 +179,20 @@ def copy(self, deep=True):
         )
         return result
 
+    @property
+    def valid_count(self) -> int:
+        """
+        Returns the number of valid geometries stored in this GeoColumn.
+        """
+        return self._meta.input_types.valid_count
+
+    def has_nulls(self) -> bool:
+        """
+        Returns True if any of the geometries stored in this GeoColumn are
+        null.
+        """
+        return self._meta.input_types.has_nulls
+
```

Comment:
Let's make sure these are not included in this PR. They belong to #894

Reply:
:+1: